sdcm/cluster.py

```diff
@@ -597,7 +610,9 @@ def reconfigure_prometheus(self, targets):
         self._write_prometheus_cfg(targets)
         self.remoter.run('sudo systemctl restart prometheus.service')
 
-    def setup_prometheus(self, targets):
+    def setup_prometheus(self, targets, metrics_cfg=True):
+        if metrics_cfg:
+            self._metrics_target = start_metrics_server()
```

Comment:
We should make sure the http server can be accessed from monitor.
If we run SCT inside an (GCE or AWS) instance that is in same region with monitor, it works.